### DISCUSSION TITLE
SQL Query Limit 5000

### DISCUSSION THREAD

**Discussion by overac** [Budibase/budibase#8444] [Help] [answered]
When connecting a MySQL database I noticed in the mysql logs that the queries are limited to 5000 returned records with the LIMIT keyword
Is there a possibility to adjust this limit ?
Thanks in advance.

**Comment by andz-bb** [comment, marked as answer]
Hey @overac, you can override this limit using the environment variable `SQL_MAX_ROWS` 👍

**Reply by melohagan** [reply]
Thanks Andy! I've added this variable into the docs as well: https://docs.budibase.com/docs/hosting-settings
